lib/scenic/statements.rb

```diff
@@ -89,6 +89,32 @@ def update_view(name, version: nil, revert_to_version: nil, materialized: false)
       end
     end
 
+    # Update a database view to a new version using `CREATE OR REPLACE VIEW`.
+    #
+    # The existing view is created/replaced using the supplied `version`
+    # parameter.
+    #
+    # Does not work with materialized views, no parameter for that is provided!
```

Comment:
I like the idea, I'll take a look at a good way to handle it and provide a nice error.